app/src/main/java/fr/free/nrw/commons/MediaDataExtractor.java

```diff
@@ -26,31 +28,49 @@
     @Inject
     public MediaDataExtractor(MediaWikiApi mwApi,
                               OkHttpJsonApiClient okHttpJsonApiClient,
-                              MediaClient mediaClient) {
+                              MediaClient mediaClient, MediaDetailInterface mediaDetailInterface) {
```

Comment:
If MediaDetailInterface is not used anywhere, we can remove that as constructor parameter

Reply:
Sorry that was placed here while investigating. I'll remove it